compiler/one-cmds/one-codegen

```diff
@@ -1,4 +1,9 @@
-#!/bin/bash
+#!/usr/bin/env bash
+''''export SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"                  # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
```

Comment:
Ah... this line is the key..

Reply:
What does ``''''`` mean in BASH?

Reply:
I guess that it is a string in Python.

Reply:
Right. `''''    #'''` becomes string in python. ' ' ' ' means `''` + `''`, two string without contents. Another example is like..
```bash
$ 'l''s'
$ "l""s"
```
and so on.